three-cards-divination/src/main/resources/openapi/api_contract.yml

```diff
@@ -0,0 +1,112 @@
+openapi: 3.0.1
+info:
+  title: OpenAPI definition
+  version: v0
+servers:
+  - url: http://localhost:8080
+    description: Generated server url
+  - url: http://localhost:5050
+    description: Docker server url
+paths:
+  /user:
+    get:
+      tags:
+        - user-controller
+      operationId: findAll
+      responses:
+        "200":
+          description: OK
+          content:
+            '*/*':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/TarotUser'
```

Comment:
**Suggestion:** It would be better to provide a more detailed description for each endpoint in the OpenAPI contract. This will help users understand what each endpoint does and how to use it. [enhancement]
```suggestion
paths:
  /user:
    get:
      tags:
        - user-controller
      operationId: findAll
      summary: Retrieve all users
      description: This endpoint retrieves all users in the system.
      responses:
        "200":
          description: OK
          content:
            '*/*':
              schema:
                type: array
                items:
                  $ref: '#/components/schemas/TarotUser'
```